libraries/exporters/src/main/java/org/eclipse/sw360/exporter/helper/ReleaseHelper.java

```diff
@@ -134,6 +134,11 @@ private void addFieldValueToRow(List<String> row, Release._Fields field, Release
                     row.add("");
```

Comment:
Do we need a second `row.add("");` here? So that we do fill the new column with an empty value?

Reply:
In case the categories field is not set then we will have to set it with empty value otherwise application will through an exception for in consistency in number of rows and columns.

Reply:
Yes, that was also my impression. And that's why I asked if we need a second of these statements here. Because the first one is for the component type, so you probably should add a second for the categories, shouldn't you?

Reply:
Yes got it thanks. You are right,Even though the chance of component being null is minimal still if component is null we will need another row.add("").I have added the same